shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-distsql/shardingsphere-readwrite-splitting-distsql-handler/src/main/java/org/apache/shardingsphere/readwritesplitting/distsql/handler/query/ReadwriteSplittingRuleQueryResultSet.java

```diff
@@ -64,9 +64,9 @@ public boolean next() {
     @Override
     public Collection<Object> getRowData() {
         ReadwriteSplittingDataSourceRuleConfiguration ruleConfig = data.next();
+        ShardingSphereAlgorithmConfiguration configuration = loadBalancers.get(ruleConfig.getLoadBalancerName());
         return Arrays.asList(ruleConfig.getName(), ruleConfig.getAutoAwareDataSourceName(), ruleConfig.getWriteDataSourceName(), Joiner.on(",").join(ruleConfig.getReadDataSourceNames()),
-                null == loadBalancers.get(ruleConfig.getLoadBalancerName()) ? null : loadBalancers.get(ruleConfig.getLoadBalancerName()).getType(),
-                PropertiesConverter.convert(loadBalancers.get(ruleConfig.getLoadBalancerName()).getProps()));
+                null == configuration ? null : configuration.getType(), null == configuration ? null : PropertiesConverter.convert(configuration.getProps()));
```

Comment:
more than 1 conditional operator in a clause is not a good practice .

Reply:
> more than 1 conditional operator in a clause is not a good practice .

@taojintianxia Thank you for your suggestion.